lua/starfall/libs_sv/prop.lua

```diff
@@ -472,7 +472,7 @@ function props_library.createSent(pos, ang, class, frozen, data)
 			local value = data[param]
 			
 			if value then
-				value = ounwrap(value)
+				value = ounwrap(value) or value
```

Comment:
What value is ounwrap returning nil? You shouldn't need this.

Reply:
Table, but looking at the sourcecode for UnwrapObject TYPE_TABLE could probably be added to safe_types.

Reply:
Ah ok that makes sense. TYPE_TABLE can't be added to safe types. ounwrap might be able to return them, but I have to first check that there's no security problems with that.

Reply:
I see, no, tables have to be explicitly allowed so the code here is correct.